apps/front/src/lib/translations/occupations.ts

```diff
@@ -4,5 +4,8 @@ export const OCCUPATIONS = {
   [OccupationTypes.FARMER]: 'Fermier',
   [OccupationTypes.CARPENTER]: 'Charpentier',
   [OccupationTypes.RETIRED]: 'Retraité',
+  [OccupationTypes.CHARCOAL_BURNER]: 'Charbonnier',
+  [OccupationTypes.GATHERER]: 'Récolteur',
+  [OccupationTypes.WOOD_CUTTER]: 'Coupeur de bois',
```

Comment:
```suggestion
  [OccupationTypes.WOODCUTTER]: 'Coupeur de bois',
```
Et du coup, un peu partout je penses